src/alembic/versions/edadb97f6502_initial_state.py

```diff
@@ -0,0 +1,385 @@
+    sa.PrimaryKeyConstraint('ac'),
+    schema='uta'
+    )
+    op.create_index(op.f('ix_uta_transcript_cds_md5'), 'transcript', ['cds_md5'], unique=False, schema='uta')
+    op.create_index(op.f('ix_uta_transcript_origin_id'), 'transcript', ['origin_id'], unique=False, schema='uta')
+    op.create_table('exon_set',
+    sa.Column('exon_set_id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('tx_ac', sa.Text(), nullable=False),
+    sa.Column('alt_ac', sa.Text(), nullable=False),
+    sa.Column('alt_strand', sa.SmallInteger(), nullable=False),
+    sa.Column('alt_aln_method', sa.Text(), nullable=False),
+    sa.Column('added', sa.DateTime(), nullable=False),
+    sa.ForeignKeyConstraint(['tx_ac'], ['uta.transcript.ac'], onupdate='CASCADE', ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('exon_set_id'),
+    sa.UniqueConstraint('tx_ac', 'alt_ac', 'alt_aln_method', name='<transcript,reference,method> must be unique'),
+    schema='uta'
+    )
+    op.create_table('exon',
+    sa.Column('exon_id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('exon_set_id', sa.Integer(), nullable=False),
+    sa.Column('start_i', sa.Integer(), nullable=False),
+    sa.Column('end_i', sa.Integer(), nullable=False),
+    sa.Column('ord', sa.Integer(), nullable=False),
+    sa.Column('name', sa.Text(), nullable=True),
+    sa.CheckConstraint('start_i < end_i', name='exon_start_i_must_be_lt_end_i'),
+    sa.ForeignKeyConstraint(['exon_set_id'], ['uta.exon_set.exon_set_id'], onupdate='CASCADE', ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('exon_id'),
+    sa.UniqueConstraint('exon_set_id', 'end_i', name='end_i_must_be_unique_in_exon_set'),
+    sa.UniqueConstraint('exon_set_id', 'start_i', name='start_i_must_be_unique_in_exon_set'),
+    schema='uta'
+    )
+    op.create_index(op.f('ix_uta_exon_exon_set_id'), 'exon', ['exon_set_id'], unique=False, schema='uta')
+    op.create_table('exon_aln',
+    sa.Column('exon_aln_id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('tx_exon_id', sa.Integer(), nullable=False),
+    sa.Column('alt_exon_id', sa.Integer(), nullable=False),
+    sa.Column('cigar', sa.Text(), nullable=False),
+    sa.Column('added', sa.DateTime(), nullable=False),
+    sa.Column('tx_aseq', sa.Text(), nullable=True),
+    sa.Column('alt_aseq', sa.Text(), nullable=True),
+    sa.ForeignKeyConstraint(['alt_exon_id'], ['uta.exon.exon_id'], onupdate='CASCADE', ondelete='CASCADE'),
+    sa.ForeignKeyConstraint(['tx_exon_id'], ['uta.exon.exon_id'], onupdate='CASCADE', ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('exon_aln_id'),
+    schema='uta'
+    )
+    op.create_index(op.f('ix_uta_exon_aln_alt_exon_id'), 'exon_aln', ['alt_exon_id'], unique=False, schema='uta')
+    op.create_index(op.f('ix_uta_exon_aln_tx_exon_id'), 'exon_aln', ['tx_exon_id'], unique=False, schema='uta')
+    # ### end Alembic commands ###
+
+    # ### custom commands to match the initial UTA database schema 1.1 ###
```

Comment:
Added these Alembic migration directives to match the existing `associated_accessions` table.